### DISCUSSION TITLE
Silence alarm

### DISCUSSION THREAD

**Discussion by iulius74** [petrleocompel/hikaxpro_hacs#109] [Q&A]
How can I silence alarm without disarming? 
In alarm control panel I only have disarm option when alarm is triggered.
Is it possible to also show a Silence button on the card? (in hik-connect you have the possibility to silence the alarm, leaving it armed).

**Comment by petrleocompel** [comment]
We cannot display silence alarm on the card. The UI is homeassistant standardized UI. 

I can look in docs if the "possibility" is there but button to mute it would be available as HA service action. So you can implement your own controls on your dashboard.

**Reply by iulius74** [reply]
service action will be ok

**Reply by iulius74** [reply]
thank you for reply

**Reply by petrleocompel** [reply]
Question is which function should be available ?

<img width="924" alt="Screenshot 2024-04-18 at 8 30 00" src="https://github.com/petrleocompel/hikaxpro_hacs/assets/9423543/ff9a9126-abf9-4d83-b7af-6322c37c393d">
<img width="953" alt="Screenshot 2024-04-18 at 8 30 25" src="https://github.com/petrleocompel/hikaxpro_hacs/assets/9423543/1b83eec4-6419-45ad-9f16-1c773ba2cf6e">
<img width="887" alt="Screenshot 2024-04-18 at 8 31 50" src="https://github.com/petrleocompel/hikaxpro_hacs/assets/9423543/39069098-7484-4ce5-b085-92464f8ce13c">

- Siren control - disable sound - but keeps alarm active
- Bypass - bypass ?
- Clear alarm - Clears alarm but keeps system secured


Not sure which one you had in mind?

**Reply by iulius74** [reply]
If the system triggers siren through a wireless relay (which is an automation for hikvision ax pro) then siren control could not be a solution. 
I think "clear alarm" is the way. The system will exit the alarm state, but remain in armed state (right?)
